src/test/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepoTest.java

```diff
@@ -121,8 +121,41 @@ void testMtDynamoDbTableDescriptionPayPerRequestIsSet() throws InterruptedExcept
     }
 
     @Test
-    void testListVirtualTableMetadata() {
+    void testListVirtualTables() {
         MtDynamoDbTableDescriptionRepo repo = mtDynamoDbTableDescriptionRepoBuilder.build();
+        List<MtCreateTableRequest> tablesCreated = createPairOfVirtualTables(repo);
+        ListMetadataResult returnedMetadatas =
+            ((MtTableDescriptionRepo)repo).listVirtualTableMetadata(new ListMetadataRequest());
+        assertEquals(new ListMetadataResult(tablesCreated, null), returnedMetadatas);
+    }
+
+    @Test
+    void testListVirtualTables_pagination() {
```

Comment:
its worth mentioning by splitting this out into 3 test cases vs 1, I added 10 seconds to the test suite. Each test case creates its own description table with a unique table name including the timestamp since epoch, which translates to at least 5 seconds per test case. 

Our test runs are already pretty slow, this is sort of a bummer, wondering how we should try conserving there.